css/styles.css

```diff
@@ -33,10 +33,100 @@ header {
   position: relative;
 }
 
-.nav {
+.main-menu {
   display: none;
 }
 
+li {
+  list-style-type: none;
+}
+
+.mobile-menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100vw;
+  background-color: #3c3a39;
+  padding: 50px 24px;
+  z-index: 2;
+  display: none;
+}
+
+.mobile-menu ul {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 24px 0;
+}
+
+.skills-details li {
+  padding: 10px 12px;
+  margin: 12px 2px 22px 13px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.languages li {
+  border: 1px solid #8993a4;
+  padding: 10px 12px;
+  background-color: #ffff3a;
+  font-family: "Inter", sans-serif;
+}
+
+.about-section-content li {
+  font-family: "Inter", sans-serif;
+  font-style: normal, sans-serif;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+}
+
+.mobile-menu ul li a {
+  text-decoration: none;
+  font-family: "Inter", sans-serif;
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 700;
+  color: #fff5e1;
+  padding-bottom: 15px;
+}
+
+.mobile-menu ul li {
+  border-bottom: 1px solid #6f6c6b;
+  width: 87.2%;
+}
+
+.mobile-menu ul li a:hover {
+  color: #e05304;
+  transition: all 0.3s;
+}
+
+.show {
+  display: block;
+}
+
+.hamburger {
+  background: url(../Images/Menu.png) no-repeat center/cover;
+  height: 16px;
+  width: 18px;
+}
+
+.close-icon {
+  background: url(../Images/icons/close.png) no-repeat center/cover;
+  width: 14px;
+  height: 12px;
+  z-index: 3;
+  position: fixed;
+  right: 4rem;
+}
```

Comment:
- [ ] [OPTIONAL] Ouch! this is so close but it seems like the cross Icon should be aligned a bit to the right and the size should be a bit bigger as well as shown in the design:
    - Figma design:
      ![image](https://user-images.githubusercontent.com/104968236/230906534-5e64da0e-119c-4ce2-beb6-7a6b600f7f35.png)
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/230906715-f7cb61c8-3a52-4e4b-8fe0-22698a9e6e6b.png)
    - Hint: you can reduce the `right` to `2rem` and see how it looks for the alignment. 😉